src/main/java/application/SessionController.java

```diff
@@ -51,6 +56,8 @@ public NewSessionResponse newFormResponse(@RequestBody String body){
             log.error(e);
         } catch (IOException e) {
             log.error(e);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
If you're going to just catch all exceptions you may as well get rid of the other 3 catch blocks. If you don't catch the exception Spring will return an error response to the client. We can also customize the error responses: http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-error-handling